### ISSUE TITLE
Opening Atom via Github 

### ISSUE THREAD

**Issue by kristasmith88** [Media-Ed-Online/intro-web-dev#18]
HELP! Please! 
Why can't I open document in Atom from Github?
<img width="1065" alt="screen shot 2017-09-09 at 11 51 36 am" src="https://user-images.githubusercontent.com/31778785/30242549-8eee4b9a-9555-11e7-8b9b-e88850ea6f5a.png">

**Comment by JustineEvansUM** [comment]
Hi, @kristasmith88. You need to create a commit message in the lower left like "added readme" and press "Publish repository" at the top.

Good luck!

**Comment by FriedmanM** [comment]
@kristasmith88 I had this same issue when following along with the directions. If you are on a Mac and in GitHub Desktop, go to the top bar on the screen and click "Repository", then click "Open in Atom".

**Comment by JustineEvansUM** [comment]
YES! I'm sorry, yes you can open Atom directory from the GitHub Desktop. I'm usually working in the reverse, but this is definitely an option. Nice catch, @FriedmanM !
